Styles/main.css

```diff
@@ -136,7 +140,7 @@ footer hr {
 }
 .paperclip{
     position: absolute;
-    right: 7px;
+    right: -7px;

```

Comment:
Using a negative number here causes the image to move to the right (as requested by the lab) instead of to the left.